src/main/java/com/dev/cinema/validation/EmailValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.dev.cinema.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<EmailValidation, String> {
+    private static final String REGEX = "^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+$";
```

Comment:
```suggestion
    private static final String VALID_EMAIL_REGEX = "^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+$";
```
regex is not informative

Reply:
done